src/lib/prefixes.ts

```diff
@@ -51,6 +50,7 @@ export default {
   sosa: 'http://www.w3.org/ns/sosa/',
   ssn: 'http://www.w3.org/ns/ssn/',
   time: 'http://www.w3.org/2006/time#',
+  unit: 'http://qudt.org/vocab/unit/',
```

Comment:
`http://qudt.org/vocab/unit/` instead of `http://qudt.org/vocab/unit#` because https://raw.githubusercontent.com/zazuko/rdf-vocabularies/d655c76bec525c5edcfc123601848b4acd735135/ontologies/unit.nq doesn't contain `http://qudt.org/vocab/unit#`